app/services/project_service.py

```diff
@@ -135,31 +158,37 @@ def _create_projects(self, fetched_projects, persisted_projects):
         persisted_project_ids = set(
             map(lambda x: x.key, persisted_projects)
         )
+
         projects_to_create = set(
             filter(lambda x: x.key not in persisted_project_ids, fetched_projects)
         )
 
-        for jira_project in projects_to_create:
+        # TODO: make this async so we don't have to wait for each project's issues
+        for jira_project in (projects_to_create or []):
             jira_project_model = Project(
                 name=jira_project.name,
                 key=jira_project.key,
             )
+
             db.session.add(jira_project_model)
 
-            fetched_ids = [x.accountId for x in
-                           self.jira_service.get_project_members(jira_project.key)]
+            # fetched_ids = [x.accountId for x in
+            #                (self.jira_service.get_project_members(jira_project.key) or [])]
 
-            for jid in fetched_ids:
-                jira_member = JIRAMember.query.filter_by(jira_member_id=jid).first()
-                if jira_member:
-                    jira_project_model.allowed_members.append(jira_member)
+            # for jid in (fetched_ids or []):
+            #     jira_member = JIRAMember.query.filter_by(jira_member_id=jid).first()
+            #     if jira_member:
+            #         jira_project_model.allowed_members.append(jira_member)
```

Comment:
Trying to see if we can/should move syncing members into it's own operation to reduce the number of things this already overloaded sync process needs to do.